### PR TITLE
Update DO example to allow for easily switching between DOs

### PR DESCRIPTION
**Pull request by bretthoerner**
I use this example as a starting point when I want to do simple testing or show someone something, and the first thing I _always_ do is make the object itself an argument.

I think hard coding `"A"` does new readers a disservice and could confuse them about DO namespaces/objects.

Most of the commentary is copy pasted from elsewhere in this document, FWIW.

### CODE REVIEW

**Comment by bretthoerner** [comment]
The failure is odd, but I don't have permissions to view: https://github.com/cloudflare/cloudflare-docs/pull/4529#issuecomment-1134624832

**Review by a-robinson — APPROVED** [review]
Yeah this makes the example much better 👍

**Comment by a-robinson** [comment]
I can't view the failure details either

**Comment by koeninger** [comment]
Talking to pages team to see if there's a way for us to view these errors, this isn't the first time it's happened.

As far as the PR goes, I agree with the idea, but we should probably update the templates too - I can do that if we're all ok with this change

**Review by koeninger — APPROVED** [review]

**Comment by koeninger** [comment]
tldr from pages team seems to be push another commit and see if it builds
